tests/e2e/config/jest.setup.js

```diff
@@ -8,10 +9,17 @@ import {
 	switchUserToAdmin,
 	switchUserToTest,
 	visitAdminPage,
+	clearLocalStorage,
 } from '@wordpress/e2e-test-utils';
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_TIMEOUT } from '../utils';
+const setDefaultOptions = require( 'expect-puppeteer' ).setDefaultOptions;
```

Comment:
Sorry, maybe that's a stupid question: can't it be imported normally? Why does it need to be `require`d? Maybe since this is odd, let's put a comment on it?
Can we at least destructure it, you think?

Reply:
Not a stupid question :D 

I copied and paste the import, I will update it 👍